### DISCUSSION TITLE
100 altını paylaşan korsanlar sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#286] [Zeka Sorusu]
100 altın paradan oluşan bir hazine var ve bir adada bulunan beş korsan bu altınları aralarında bölüşecekler. Ganimet şu kurala göre paylaşılacak: En yaşlı korsan, nasıl bölüşebilecekleriyle ilgili bir teklifte bulunacak ve bu teklif oylanacak. Eğer korsanların en az yarısı kabul ederse, altınlar o şekilde bölüşülecek. Aksi taktirde, en yaşlı korsanı öldürüp aynı şekilde devam edecekler. Geriye kalanların en yaşlısı kendi teklifini sunacak ve oylama sonucunda aynı kurala göre ya en yaşlı korsan öldürülecek yada altınlar paylaşılacak. Bir teklif üzerinde anlaşılana kadar bu böyle devam edecek.
En yaşlı korsan siz olsaydınız nasıl bir bölüşme önerirdiniz?
(Korsanların hepsi olabildiğince mantıklı ve açgözlü davranıyorlar, ayrıca hiç biri ölmek istemiyor)

**Comment by FatihOnder81** [comment]
Eğer herkese eşit dağıtırsak korsanlar açgözlü olduğu içün bunu kabul etmeyebilir bende ölmek istemediğim için kendime 0 altın alırım ve topu bir sonraki korsana atarım

**Comment by yukseel** [comment]
Kendime 98 altın alırım geriye kalan 2 altını da en yaşlı olan 2 kişiye vermek için hazırlarım. 2 korsan garanti olarak kabul edecektir çünkü açgözlü oldukları için iki arkadaşının almaması onları motive eder, böylece hem kendim en fazla parayı alırım hem de 2 korsan garanti olduğu için kurtulurum

**Comment by devrimsuzer** [comment]
Kendime 22 altın alır kendimden sonraki en yaşlı korsana 21 altın veririm ondan sonraki en yaşlı korsana 20 altın verir sonrakine 19 altın
sonrakine de 18 altın veririm.Bu sayede hem ölmem hem karda olurum.Ölmem çünkü diğerleri en az 1 kişiden daha fazla altın alır bu yüzden
sadece biri itiraz eder.(en genç korsan)
